packages/rum-core/src/boot/rumPublicApi.spec.ts

```diff
@@ -418,10 +411,19 @@ describe('rum entry', () => {
       setupBuilder.cleanup()
     })
 
+    it('should do nothing before init', () => {
```

Comment:
I expected to follow the same pattern as other APIs and set the timing even if it called before init. Is there a reason not to do this?

Reply:
I thought it would not make sense but it could 🤔, I'll have a look